freg-gateway-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/freg/config/FregGatewayConfig.java

```diff
@@ -0,0 +1,26 @@
+package no.difi.meldingsutveksling.serviceregistry.freg.config;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.ClientHttpRequestInterceptor;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+@RequiredArgsConstructor
+public class FregGatewayConfig {
+    public static final String CLIENT_ID = "Client-Id";
+
+    @Bean(name = "fregGatewayRestTemplate")
+    public RestTemplate fregGatewayRestTemplate() {
+        RestTemplate restTemplate = new RestTemplate();
+        return restTemplate;
+    }
+
+    protected ClientHttpRequestInterceptor clientIdRequestInterceptor() {
+        return (request, body, execution) -> {
+            request.getHeaders().add(CLIENT_ID, "TestClient");
+            return execution.execute(request, body);
+        };
```

Comment:
Skulle denne fjernes, eller skal den være der?

Reply:
Var aldri brukt, se i minid-new-user-backend for eksempler på hvordan det brukes om vi trenger en interceptor.